contrib/Docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.12
+
+# ---------------- #
+#   Installation   #
+# ---------------- #
+
+# Install and setup all prerequisites
+RUN apk add --no-cache gcc g++ python3 py3-pip python3-dev supervisor                &&\
+    rm -rf /var/cache/apk/*                                                          &&\
+    pip3 install numpy                                                               &&\
```

Comment:
Yes we can copy requirements.txt to Dockerfile and source that, but why we should do that? because pip3 command will install necessary packages.

Reply:
Will `pip3 install haproxystats` bellow install all the dependencies declared in https://github.com/unixsurfer/haproxystats/blob/master/requirements.txt? Since we don't set the dependencies in `setup.cfg` I don't think that  `pip3 install haproxystats` will pull down the necessary dependencies. I could be very much wrong here.

Reply:
I will update the Dockerfile and push it again with your idea.

Reply:
The requirements.txt  has been added into Dockerfile.